src/library/scala/beans/ScalaBeanInfo.scala

```diff
@@ -32,15 +31,9 @@ abstract class ScalaBeanInfo(clazz: java.lang.Class[_],
   override def getPropertyDescriptors() = pd
   override def getMethodDescriptors() = md
 
-  // override def getAdditionalBeanInfo() = Array(Introspector getBeanInfo clazz.getSuperclass)
-
   private def init() {
-    var i = 0
```

Comment:
I do agree that the previous code may be more expensive, due to the filter etc. on the methods.  Maybe that should be sped up, if this is at all important.